study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/task/CircadiaUpdateInstanceCreationEvent.java

```diff
@@ -0,0 +1,85 @@
+package org.broadinstitute.ddp.studybuilder.task;
```

Comment:
Does this patch/task need to go into patch-log.conf ?

Reply:
No, it doesn't. Basic configuration is updated too, as usual